### PR TITLE
Added layout header / footer localisation.

### PR DESCRIPTION
**Pull request by dbgoaco**
[DP-788](https://noticingsystem.atlassian.net/jira/software/projects/DP/boards/22?selectedIssue=DP-788)

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -69,11 +69,11 @@
                     </div>
                     <div class="app-header-layout__column">
                         <a href="@homeLink" class="govuk-header__link govuk-header__service-name">
-                            Find a Tender
+                            @StaticTextResource.Header_Service_Name
                         </a>
                         <nav aria-label="Menu" class="govuk-header__navigation">
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml:74]:
```suggestion
                        <nav aria-label="@StaticTextResource.Header_Menu" class="govuk-header__navigation">
```

File: Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -160,25 +160,25 @@
                     <h2 class="govuk-visually-hidden">Support links</h2>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml:160]: "Support links" needs translating here

File: Services/CO.CDP.Localization/StaticTextResource.resx [hunk 3]
```diff
@@ -174,4 +174,49 @@
   <data name="Global_Continue" xml:space="preserve">
     <value>Continue</value>
   </data>
+  <data name="Header_Service_Name" xml:space="preserve">
```
Comment (andymantell) [inline Services/CO.CDP.Localization/StaticTextResource.resx:177]:
```suggestion
  <data name="Header_ServiceName" xml:space="preserve">
```

Could we tweak the structure of the keys please. I kind of see the underscore as an indicating of increasing specificity within the overall hierarchy of keys. And within each segment of the key CamelCasing is enough to make it readable.

For example, the key `OrganisationRegistration_EnterOrganisationName_Heading`

is in the `OrganisationRegistration` section, on the `EnterOrganisationName` page and it addresses the `Heading`

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml (hunk 2 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml:160]: Missed that, done.

File: Services/CO.CDP.Localization/StaticTextResource.resx (hunk 3 above)
Comment (dbgoaco) [reply Services/CO.CDP.Localization/StaticTextResource.resx:177]: Updated structure of the keys. Think they are ok now.

File: docs/development/cookbooks/02-how-to-localize-ui-text.adoc [hunk 4]
```diff
@@ -16,7 +16,7 @@ There are two separate resource files - `StaticTextResource` which is text that
 
 Generally speaking, keys are either "global" in that they are not unique to a particular section and may appear anywhere on the service - for example `Global_Continue` represents the word `Continue` on the buttons that appear across the site. Or they can be namespaced to a specific section of the service such as `BasicInformation_SectionTitle`.
 
-Use Underscores_To_Separate_Words and not dots or hyphens. This ensures that the key is the same whether you are using the strongly typed `StaticTextResource` class to access keys like `StaticTextResource.Global_Back`, or if you are using `@inject IHtmlLocalizer<StaticTextResource> Localizer` to access keys like `Localizer["Global_Back"]`
+Use Underscores_To_Separate_Words and not dots. This ensures that the key is the same whether you are using the strongly typed `StaticTextResource` class to access keys like `StaticTextResource.Global_Back`, or if you are using `@inject IHtmlLocalizer<StaticTextResource> Localizer` to access keys like `Localizer["Global_Back"]`
```
Comment (andymantell) [inline docs/development/cookbooks/02-how-to-localize-ui-text.adoc:19]:
```suggestion
Use Underscores_To_Separate and not dots or hyphens. This ensures that the key is the same whether you are using the strongly typed `StaticTextResource` class to access keys like `StaticTextResource.Global_Back`, or if you are using `@inject IHtmlLocalizer<StaticTextResource> Localizer` to access keys like `Localizer["Global_Back"]

Underscores should be used to separate "sections" of the key. And CamelCase should be used within those sections. So for example the key `OrganisationRegistration_EnterOrganisationName_Heading`  represents a `Heading` within the `EnterOrganisationName` page in the `OrganisationRegistration` section of the service.

"Section" could be anything from site section, page, area of the page, specific type of org that it applies to - whatever makes sense in the context of the key you are naming. Broadly though, each underscore represents an increase in specificity.
```

**Review by andymantell — APPROVED** [review]

**Review by dpatel017 — APPROVED** [review]

**Pull request merged**